src/orion/benchmark/task/profet/xgboost_test.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
+import pytest
+from .xgboost import XgBoostTask
+from .profet_task import MetaModelTrainingConfig
+
+
+@pytest.mark.skip(reason="Take WAY too long to run.")
+def test_kwargs_fix_dimension_xgboost(profet_train_config: MetaModelTrainingConfig):
```

Comment:
Same comments as for the other test. (https://github.com/Epistimio/orion/pull/632/files#r690463376 and https://github.com/Epistimio/orion/pull/632/files#r690464321)